modpack_format/update_format.md

```diff
@@ -0,0 +1,32 @@
+# Update Format
+The Update Format is a simple format that can be used to specify all available versions of a mod or modpack. It's most useful in the `updates` field in the modpack format, which allows launchers to let users update modpacks automatically.
+
+## Fields
+
+### `formatType`
+Set to `modipUpdates` (TODO: Change name?)
+
+---
+
+### `formatVersion`
+Set to `1.0.0`.
+
+---
+
+### `id`, `name`, `summary`, `description`
+This fields all behave exactly like the respective fields in the modpack format.
+
+---
+
+### `versions`
+This is an array containing version objects. Each version object is comprised of the following:
+
+#### `name`
+Human-readable name of the version.
+
+#### `releaseDate`
+The release date of the version, following ISO 8601. See the `releaseDate` field in the modpack format for more information.
+
+#### `downloads`
+An array listing URLs where the modpack zip can be downloaded.
+(TODO: zip or no zip?)
```

Comment:
Whichever one is normally used to distribute the modpack. Realistically, most launchers won't care about your extensive internet infrastructure. They will download, and install, simple as that

Reply:
Maybe you can add a separate JSON download (which points to a JSON containing download URLs), just in case both the launcher and the publisher are willing to support partial downloads

Reply:
(note: last suggestion is not compatible with automatically copying overrides)

Reply:
I suspect zip will be the most common way to distribute it. I'm a bit confused what you mean by json endpoint though - what do the download URLs point to, modpack json files (index.modip.json)?

Reply:
What I mean is, there could be an optional extension for fast updates where the launcher downloads the JSON file by itself, then downloads the files that have changed, instead of downloading the entire modpack again including the parts that haven't changed.

Reply:
Got it. That sounds useful. However there's an issue handling overrides if they're included inside the zip in a directory (which I have recently PR'd). Maybe there should be a hash of the overrides directory stored, and it can be compared? I'm not sure.

Reply:
Resolved, suggestion wasn't compatible with overrides dir